static/js/zamboni/reviewers.js

```diff
@@ -1,3 +1,5 @@
+NO_ATTACHMENT_ACTIONS = ['disable_addon', 'enable_addon'];
```

Comment:
Can you add a comment of where the source of these strings is, since they are literals.